### PR TITLE
updates gee datasets to 2020 and fixes bug with reprojection for lpks plots

### PR DESCRIPTION
**Pull request by gdaldegan**

### CODE REVIEW

File: gee/landpks/src/main.py [hunk 1]
```diff
@@ -366,7 +366,7 @@ def base_image(year, geojson, lang, gc_client, metadata):
 
     # Reproject L8 image so it can retrieve data from every latitute 
     # without deforming the aoi bounding box
-    l8sr_mosaic_bands = l8sr_y.reproject(scale=30, crs='EPSG:3857')
+    map_l8sr_mosaic = map_l8sr_mosaic.reproject(**({'crs':'EPSG:3857','scale':30}))
```
Comment (azvoleff) [inline gee/landpks/src/main.py:369]: There is no need for the ** notation here to set the parameters from a dictionary (it is kind of a holdover from Javascript). In Python this line can just be:

`map_l8sr_mosaic = map_l8sr_mosaic.reproject(crs='EPSG:3857', scale=30)`

File: gee/landpks/src/main.py [hunk 2]
```diff
@@ -418,7 +418,7 @@ def greenness(year, geojson, lang, gc_client, metadata):
     
     # Reproject ndvi mean image so it can retrieve data from every latitute 
     # without deforming the aoi bounding box
-    ndvi_mean_reproject = ndvi_mean.reproject(scale=30, crs='EPSG:3857')
+    map_mean_mosaic = map_mean_mosaic.reproject(**({'crs':'EPSG:3857','scale':30}))
```
Comment (azvoleff) [inline gee/landpks/src/main.py:421]: Same comment here - can you remove the ** notation?

File: gee/landpks/src/main.py [hunk 3]
```diff
@@ -479,7 +479,7 @@ def greenness_trend(year_start, year_end, geojson, lang, gc_client, metadata):
             .visualize(**{'palette': ['black'], 'opacity': 1})]) \
             .mosaic()
     
-    # Reproject ndvi mean image so it can retrieve data from every latitute 
+    # Reproject ndvi mean image so it can retrieve data from every latitude 
     # without deforming the aoi bounding box
     map_trnd_mosaic = map_trnd_mosaic.reproject(**({'crs':'EPSG:3857','scale':30}))
```
Comment (azvoleff) [inline gee/landpks/src/main.py:484]: And I hadn't noticed it before, but can you fix this ** notation also?

**Review by azvoleff — CHANGES_REQUESTED** [review]
Just a few quick changes to make the code more in line with the other GEE code (there are a few things that are more reflective of JavaScript than of Python) - can you fix these and then submit the updated pull request?

File: gee/landpks/src/main.py (hunk 3 above)
Comment (gdaldegan) [reply gee/landpks/src/main.py]: kwarg notations removed for all reprojections

**Pull request closed without merging**